application/controllers/admin/useraction.php

```diff
@@ -305,7 +302,7 @@ public function deleteFinalUser($result, $transfer_surveys_to)
             die();
         }
 
-        $extra = "<br />".sprintf(gT("User '%s' was successfully deleted."), $postuser)."<br /><br />\n";
+        $extra = "<br />".sprintf(gT("User '%s' was successfully deleted."), flattenText($fields['full_name']))."<br /><br />\n";
```

Comment:
Why not Chtml::encode ?

Reply:
I tried to do the changes as minimal invasive as possible, i.e. I kept flattenText() from the removed lines above. If you think Chtml::encode() is better in this case, I can change it.

To tell the truth I'm not sure how LimeSurvey uses flattenText() and that is the difference because in one place I found it used, other similar places uses hardcoded htmlspecialchars() and others use Chtml::encode().

Reply:
`Chtml::encode` == `htmlspecialchars` ;)

https://www.yiiframework.com/doc/api/1.1/CHtml#encode-detail

I totally understand your pov about « changes as minimally» but clearly here : a Chtml::encode seems better.

Reply:
Done.